core/src/main/java/io/opencensus/stats/NoopStats.java

```diff
@@ -88,10 +99,28 @@ public StatsRecorder getStatsRecorder() {
     static final StatsRecorder INSTANCE = new NoopStatsRecorder();
 
     @Override
-    public void record(TagContext tags, MeasureMap measureValues) {
+    public StatsBuilder newStatsBuilder(TagContext tags) {
```

Comment:
nit: might be better to have a method like `StatsBuilder.withExplicitTagContext(TagContext tags)` instead of passing TagContext to the builder method? I think if there's a parameter to newBuilder(), usually that parameter should be required, which is not the case here.

Reply:
As we discussed offline, the API should make the distinction between the two use cases more clear.  I renamed `StatsRecorder.newStatsBuilder(TagContext)` to `newStatsBuilderWithExplicitTagContext`.